pkgs/development/libraries/check/default.nix

```diff
@@ -11,11 +15,16 @@ stdenv.mkDerivation rec {
     sha256 = "02m25y9m46pb6n46s51av62kpd936lkfv3b13kfpckgvmh5lxpm8";
   };
 
-  # Test can randomly fail: https://hydra.nixos.org/build/7243912
-  doCheck = false;
+  # fortify breaks the libcompat vsnprintf implementation
+  hardeningDisable = lib.optionals (stdenv.hostPlatform.isMusl && (stdenv.hostPlatform != stdenv.buildPlatform)) [ "fortify" ];
+
+  nativeBuildInputs = [ autoreconfHook pkg-config ];
```

Comment:
why the autoreconfHook? pkgsStatic.check also builds without it

Reply:
Ah, that was required when the fix involved changing the configure.ac file since it invalidated the `configure` script included with the release.

I can remove it unless there is an upside to generating the autoconf script within the nix sandbox?

Reply:
Yeah, please remove it